src/cpp/src/image_generation/models/clip_text_model.cpp

```diff
@@ -97,6 +97,7 @@ CLIPTextModel& CLIPTextModel::compile(const std::string& device, const ov::AnyMa
     } else {
         compiled_model = core.compile_model(m_model, device, properties);
     }
+    ov::genai::utils::print_compiled_model_properties(compiled_model, core);
```

Comment:
why only clip model is printed? what's about all other models in other pipelines?

Reply:
Updated